OxfordBodleian/Juel-Jensen/BDLaethg28.xml

```diff
@@ -117,13 +117,13 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                     </list>
                                 </collation>
 
-                                <condition key="deficient">The original sewing is broken and it has been repaired by using a different kind of thread. <!--A brown liquid
+                                <condition key="deficient"><!--A brown liquid
                                      was spilled on the manuscript, see especially on <locus
                                         from="15r" to="16v"/> I think that a different kind of ink, a brownish one, has been used to draw
                                       the figures on f. 15v and f. 16v later damaged by humidity. The same ink is on f. 1v, 2r, 9r, 9v, 10r, 10v, 11r, 11v.-->
-                                        Smudged ink on many folios see especially on <locus
-                                           from="15r" to="16v"/>.</condition>
-
+                                        Smudged ink on many folios, see especially on <locus from="15r" to="16v"/>. Insect droppings and humidity stains
+                                        on most leaves. The original sewing is broken with the risk
```

Comment:
Did we agree to use "folio/folios" everywhere instead of "leaf/leaves"? Can this be changed automatically at a later stage or should be done now?